data/categories.txt

```diff
@@ -3751,7 +3751,7 @@ vi:Nhà vệ sinh
 zh-Hant:2洗手間|wc
 el:μπάνιο|χώρος ανάγκης|αποχωρητήριο|wc
 sk:3Toalety|wc
-fa:توالت
+fa:توالت|دستشویی
```

Comment:
It's better to remove this line: both synonyms will be added by the group term (@toilet).